generator/src/Test/Pos/Block/Logic/Mode.hs

```diff
@@ -173,10 +175,11 @@ withTestParams TestParams {..} = withGenesisSpec _tpStartTime coreConfiguration
 -- The fields are lazy on purpose: this allows using them with
 -- futures.
 data TestInitModeContext = TestInitModeContext
-    { timcDBPureVar   :: DBPureVar
-    , timcSlottingVar :: TVar SlottingData
-    , timcSystemStart :: !Timestamp
-    , timcLrcContext  :: LrcContext
+    { timcDBPureVar        :: DBPureVar
+    , timcSlottingVar      :: TVar SlottingData
+    , timcSlottingStateVar :: SimpleSlottingStateVar
+    , timcSystemStart      :: !Timestamp
+    , timcLrcContext       :: LrcContext
```

Comment:
Is there a reason why the timestamp is the only strict field in the record?

Reply:
I only added `timcSlottingStateVar`, the other lines were only modified to comply with styling rules. Looking at [Test.Pos.Block.Logic.Mode](https://github.com/input-output-hk/cardano-sl/blob/coot/csl-2346/generator/src/Test/Pos/Block/Logic/Mode.hs#L249) only the `timcSlottingVar` and `timcLrcContext` needs to be lazy.